### ISSUE TITLE
`auto shipit` src refspec $branch does not match any

### ISSUE THREAD

**Issue by natalyna** [intuit/auto#603]
**Describe the bug**
Command `auto shipit` occures
```
Error: Running command 'git' failed

error: failed to push some refs to 'git@github.com:dodopizza/dodo-b2b-ui.git'

    at ChildProcess.<anonymous> (C:\repo\dodo-b2b-ui\node_modules\@auto-it\core\dist\utils\exec-promise.js:98:36)
    at ChildProcess.emit (events.js:198:13)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
```

**To Reproduce**
Win 10, powershell or bash
auto v7.8.0
Whithout .autorc config

GH_TOKEN has full access

**Desktop:**

- OS: Windows 10
- PowerShell
Name:                          Value:                                                                                                                                                                                                                                                       
PSVersion                      5.1.18362.145
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.18362.145
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

**Comment by hipstersmoothie** [comment]
@adierkens you may have to field this one since I don't have any windows machines

**Comment by hipstersmoothie** [comment]
@adierkens have you ever run auto from a windows machine?

**Comment by adierkens** [comment]
@hipstersmoothie Yeah I have in the past. I may not be able to get to this until the weekend, but I can try and repro

**Comment by adierkens** [comment]
@natalyna Can you test using `auto@7.11.1-canary.613.7997.0` (the canary version of #613)? 
I was able to reproduce the issue in powershell and it all seems to be working using that version:

![image](https://user-images.githubusercontent.com/13004162/67168486-e9b36d80-f359-11e9-9b20-eff39f8ac761.png)

**Comment by adierkens** [comment]
:rocket: Issue was released in v7.12.1 :rocket:
